packages/cascara/src/structures/Admin/AdminDrawer.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import pt from 'prop-types';
+import { Role } from 'reakit/Role';
 import styles from './Admin.module.scss';
+import classnames from 'classnames/bind';
+
+const cx = classnames.bind(styles);
 
 const propTypes = {
   children: pt.oneOfType([pt.node, pt.array]),
+  className: pt.string,
 };
 
-const AdminDrawer = ({ children, ...rest }) => {
+const AdminDrawer = ({ children, className, ...rest }) => {
   return (
-    <div {...rest} className={styles.Drawer}>
+    <Role {...rest} className={cx(className, 'Drawer')}>
```

Comment:
@rociobaena It's nice to see that we are swapping these, it's a much more powerful way of applying css classes 👍